openedx/features/course_experience/tests/views/test_course_outline.py

```diff
@@ -132,6 +135,18 @@ def test_outline_details(self):
                         self.assertContains(response, sequential.format)
                     self.assertTrue(sequential.children)
 
+    def test_reset_course_deadlines(self):
+        course = self.courses[0]
+        enrollment = CourseEnrollment.objects.get(course_id=course.id)
+        ScheduleFactory(
+            start_date=timezone.now() - datetime.timedelta(1),
+            enrollment=enrollment
+        )
+        url = '{}{}'.format(course_home_url(course), 'reset_deadlines')
```

Comment:
`course_home_url()` has a trailing slash, but reset deadlines does not? Just odd seeing the formatting like that.